templates/github/.github/workflows/ci.yml.j2

```diff
@@ -27,46 +22,46 @@ jobs:
   {% include pre_job_template.path %}
   {%- endif %}
   {%- if check_commit_message or lint_requirements %}
-
   ready-to-ship:
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-latest"
     steps:
       {{ checkout(depth=0, path=plugin_name) | indent(6) }}
       {{ setup_python() | indent(6) }}
       {%- if check_commit_message %}
-      - name: Install requirements
-        run: pip3 install github
-      - name: Check commit message
+      {{ install_python_deps(["requests", "pygithub"]) | indent(6) }}
```

Comment:
We install missing packages in the scripts themselves; the question is, do we need this^?

https://github.com/pulp/plugin_template/blob/f2059547f44036fef490e00863e058ce85881076/templates/github/.github/workflows/scripts/check_commit.sh.j2#L13

Reply:
I thought i removed that.

Reply:
Actually, i did.